.github/pymeasure_python312.yml

```diff
@@ -3,24 +3,27 @@ channels:
   - conda-forge
 dependencies:
   - cloudpickle=1.6.0
-  - numpy=1.26.4
-  - pandas=2.2.0
-  - pint=0.18
-  - pyqt=5.15.9
+  - numpy=2.1.2
+  - pandas=2.2.3
+  # - pint=0.24
+  # - pyqt=5.15.9  # not for py313 compiled
   - pyqtgraph=0.12.4
   - pyserial=3.4
   - pyvisa=1.14.1
-  - pyzmq=25.1.2
-  - qt=5.15.8
+  - pyzmq=26.2.0
+  # - qt=5.15.8
 # Development dependencies below
   - pytest-qt=4.2.0
   - pytest-runner=5.2
   - pytest=7.2.0
   - pytest-cov=4.1.0
-  - pyvisa-sim==0.5.1
+  - pyvisa-sim==0.6.0
   - flake8=6.0.0
   - setuptools_scm # don't pin, to get newest features
-  - sphinx=5.3.0
-  - sphinx_rtd_theme=1.2.2
+  - sphinx=8.1.3
+  - sphinx_rtd_theme=3.0.0
 #  pip is currently not needed, but the recommended tool when packages that are unavailable on conda are required
-#  - pip  # don't pin, to gain newest conda compatibility fixes
+  - pip  # don't pin, to gain newest conda compatibility fixes
+  - pip:
+    - pyqt5
+    - pint
```

Comment:
what's wrong with pint? I can't check myself atm because of network issues.

Reply:
weird, it seems like it should be available: https://conda-metadata-app.streamlit.app/?q=conda-forge%2Fnoarch%2Fpint-0.24.3-pyhd8ed1ab_0.conda